.github/workflows/deploy-zeit-production.yml

```diff
@@ -58,14 +75,23 @@ jobs:
     steps:
       - uses: actions/checkout@v1 # Get last commit pushed - XXX See https://github.com/actions/checkout
       - name: Resolving deployment url from Zeit
-        # The following workflow is:
-        #  - getting all deployments data (by using the scope in `now.json`)
-        #  - then we get the last url (in Node.js it corresponds as `response.deployments[0].url`
-        #  - and then we remove the `"` character to pre-format url
-        # We need to set env the url for next step, formatted as `https://${url provided by API}`
+        # Workflow overview:
+        #   - Resolve customer to deploy from github event input (falls back to resolving it from now.json file)
+        #   - Resolve $ZEIT_DEPLOYMENT
+        #     - Fetch all deployments data (by using the scope in `now.json`)
+        #     - Resolve the last url (from `response.deployments[0].url`)
+        #     - Remove the `"` character to pre-format url
+        # We need to set env the url for next step, formatted as `https://${$ZEIT_DEPLOYMENT}`
         run: |
           apt update -y >/dev/null && apt install -y jq >/dev/null
```

Comment:
@Demmonius Could you explain what this does and why we need it? It's not documented within the file AFAIK.